lib/matplotlib/backends/backend_qt.py

```diff
@@ -740,16 +740,29 @@ def _icon(self, name):
         Construct a `.QIcon` from an image file *name*, including the extension
         and relative to Matplotlib's "images" data directory.
         """
-        # use a high-resolution icon with suffix '_large' if available
-        # note: user-provided icons may not have '_large' versions
+        # Check for SVG first, then fall back to PNG
         path_regular = cbook._get_data_path('images', name)
+
+        # Try SVG first
+        svg_path = path_regular.with_suffix('.svg')
```

Comment:
I don't think we need to be defensive about this, we should expect the svg files to always be there.